test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -1371,6 +1371,197 @@ dir -Recurse `
             $res.CompletionMatches | Should -HaveCount $expectedCompletions
             $res.CompletionMatches[0].CompletionText | Should -BeExactly 'about_Splatting'
         }
+
+        It 'Should complete help keywords' {
+            $TestString = @'
+<#
+.^
+#>
+'@
+            $CursorIndex = $TestString.IndexOf('^')            
+            $res = TabExpansion2 -cursorColumn $CursorIndex -inputScript $TestString.Remove($CursorIndex, 1)
```

Comment:
You should write a function to factor this out in every case.

Ideally you could turn these tests into a test cases parameter to `It` with the completions you expect.

The tests should be specific though. Rather than looking at `$rest.CompletionMatches.Count`, the test should specifically expect completion output.

Reply:
Example:

https://github.com/PowerShell/PowerShell/blob/6d4420f4b96fefa92b4a03080c1b36e3bb064bf0/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1#L231-L237

Reply:
Thanks, I didn't realize you could make testcases with dynamic names like "<inputStr>".